src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -89,10 +89,21 @@
       <Link>Common\System\HResults.cs</Link>
     </Compile>
   </ItemGroup>
```

Comment:
The `TargetsWIndows` part of the condition is redundant, as uapaot and uap, will always be true